src/main/java/loanbook/logic/commands/SetEmailCommand.java

```diff
@@ -0,0 +1,80 @@
+package loanbook.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static loanbook.commons.util.CollectionUtil.requireAllNonNull;
+
+import loanbook.commons.core.Messages;
+import loanbook.logic.CommandHistory;
+import loanbook.logic.commands.exceptions.CommandException;
+import loanbook.model.Model;
+
+/**
+ * Set user's email to the app.
+ */
+public class SetEmailCommand extends Command {
+
+    public static final String COMMAND_WORD = "setemail";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Set your email to the app. "
+            + "Please note that the default email is <default>!\n"
+            + "Parameter: OLDEMAIL NEWEMAIL\n"
+            + "Example: " + COMMAND_WORD + " default myemail@gmail.com";
+
+    public static final String MESSAGE_SUCCESS = "Your email is set successfully!";
+
+    private final String oldEmail;
+    private final String newEmail;
+
+    /**
+     * Creates an SetEmailCommand to set user's {@code Email} according to the {@code newEmail} provided.
+     */
+    public SetEmailCommand(String oldEmail, String newEmail) {
```

Comment:
Can we just require a new email and a password for change? No need to know what the previous email is.

Reply:
I think we need old email? Bcs like when we want to change the password of some accounts, we need to know the old password or receive a certification email / SMS. For this case, we have no certification email/SMS function, so if we just change user email without knowing old email, it might not be safe. And that's why I want the checkemail function to show censored email.

Reply:
Can we require a password for this too? Code can be found from `delete`.

Reply:
I think it is not necessary to add a password in this command. I think security is alr guaranteed, bcs the user's email is censored. And thank you so much for your comments.